.vsts-ci/misc-analysis.yml

```diff
@@ -60,7 +61,7 @@ phases:
       Write-Host "##vso[results.publish type=NUnit;mergeResults=true;runTitle=Common Tests;publishRunAttachments=true;resultFiles=$path;]"
       if($results.TotalCount -eq 0 -or $results.FailedCount -gt 0)
       {
-        throw "markdown link tests failed"
+        throw "Common tests failed"
```

Comment:
"Common tests failed" is a little confusing without knowing that means the tests under `./test/common` folder. Maybe it's better say `Tests in './test/common' failed`.

Reply:
Changed to `markdown`